### PR TITLE
Recover service statuses for client channels

### PR DESCRIPTION
**Pull request by ababo**
Partially resolves BV-1225

### CODE REVIEW

File: data/account_test.go [hunk 1]
```diff
@@ -1,25 +1,18 @@
+// +build !nodatatest
```
Comment (furkhat) [inline data/account_test.go]: redundant build tag, *_test files ignored by default.

File: data/recover.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package data
+
+import (
+	"fmt"
+
+	"gopkg.in/reform.v1"
+)
+
+// Recover ensures data consistency after unexpected controller's exit.
```
Comment (furkhat) [inline data/recover.go]: Please extend this docstring. Write how you do it
```
Recover ensures data consistency after unexpected controller's exit by moving all semi active services to suspended state```
```

Comment (ababo) [reply data/recover.go]: @furkhat updated

File: data/account_test.go (hunk 1 above)
Comment (ababo) [reply data/account_test.go]: @furkhat Will discuss that with the team.

Comment (ababo) [reply data/account_test.go]: @furkhat removed them

**Pull request merged**